src/People/People.js

```diff
@@ -0,0 +1,79 @@
+import axios from 'axios'
+import React, { Component } from 'react'
+import "./people.css"
+
+export default class People extends Component {
+    constructor(){
+        super()
+        this.state={
+            searchperson: '',
+            currentperson: {},
+            namepers: '',
+            isError: false
+
+        }
+    }
+
+    handleCahnge= (e)=>{
+        this.setState({
+            searchperson: e.target.value
+        })
+    }
+
+    handleinput = async (e)=>{
+        e.preventDefault()
+        const {searchperson}=this.state
+
+        try{
+            const {data} = await axios.get('https://ghibliapi.herokuapp.com/people/')
+            const person = data.find((eve)=> eve.name == searchperson)
+            if(person === undefined){
+               this.setState({
+                searchperson: '',
+                currentperson: {},
+                namepers: "",
+                isError: true
+               }) 
+            }
+            this.setState({
+                currentperson: person,
+                searchperson: '',
+                namepers: person.name,
+                isError: false
+            })
```

Comment:
This setState will happen even if `person` is undefined. Consider putting this inside an else block?